pkg/volume/emptydir/empty_dir.go

```diff
@@ -45,7 +45,8 @@ import (
 // from the group attribute.
 //
 // https://issue.k8s.io/2630
-const perm os.FileMode = 0777
+var defaultPerm os.FileMode = 0777
+var stickyBitPerm os.FileMode = 01777
```

Comment:
We could instead store the sticky bit as `0o1000` and use a  bitewise operation to set it on top of the default perm, that way, if the default changes we don't need to change 2 vars.

Something like this https://goplay.tools/snippet/GcdlAf1gyi5

Reply:
Good idea @luisdavim, I just pushed a change that does this